org.kicad.KiCad.ODBCDriver.sqliteodbc.yml

```diff
@@ -0,0 +1,49 @@
+id: org.kicad.KiCad.ODBCDriver.sqliteodbc
```

Comment:
the extension point in beta is `org.kicad.KiCad.Nightly.ODBCDriver`

this is why this is failing.

Reply:
Ouch, thanks. I will prepare another PR over at the main repo then. The other Para can be left alone for now then... (#3819 and #3820)

Reply:
PR https://github.com/flathub/org.kicad.KiCad/pull/134 has been merged and the beta build is done.